ports/sqlcipher/portfile.cmake

```
@@ -0,0 +1,84 @@
+vcpkg_fail_port_install(ON_LIBRARY_LINKAGE "static" ON_TARGET "UWP" "OSX" "Linux")

```

```suggestion
vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)

vcpkg_fail_port_install( ON_TARGET "UWP" "OSX" "Linux")
```
We suggest to use `vcpkg_check_linkage()` to express unsupported for static or dynamic library.

I also noticed that you add `!arm` in vcpkg.json file. If it doesn't support arm, please also add `arm `in unsupported messages.

---

Optimistically I'm hoping arm is working, so not excluded any more.